packages/flutter_tools/.analysis_options

```diff
@@ -7,12 +7,16 @@
 analyzer:
   language:
     enableSuperMixins: true
-  strong-mode: false #TODO(pq): enable once strong-mode runs clean
+  strong-mode: true
   errors:
     # we allow overriding fields (if they use super, ideally...)
     strong_mode_invalid_field_override: ignore
     # we allow type narrowing
     strong_mode_invalid_method_override: ignore
+    strong_mode_static_type_error: ignore
+    strong_mode_down_cast_composite: ignore
+    type_argument_not_matching_bounds: ignore
```

Comment:
you caught me. 😉

I just used that as a temporary hack to work around a bug I hadn't quite figured out while watching alpha go last night; I needed to get this in before Adam's big patch lest I get into conflict hell.

I actually have this fixed locally already now. It'll be in my next patch which finishes the type work.